lieer/local.py

```diff
@@ -413,6 +419,24 @@ def messages_to_gids (self, msgs):
 
     return (messages, gids)
 
+  def nm_messages_to_gids (self, msgs):
```

Comment:
Please use the existing function, or share code in some way.

Reply:
Okay, using the existing function might affect the performance if the number of new messages is large. Then, making a new array of messages is completely unnecessary and can add to the processing load. Should I still resort to code directly than to make a new function?

Reply:
I have still used the function that I have added, let me know if you want me to include the code directly in gmailieer.py instead.